autoload/lsc/diff.vim

```diff
@@ -27,15 +27,47 @@ function! lsc#diff#compute(old, new) abort
   return result
 endfunction
 
+let s:has_lua = has('lua') || has('nvim-0.4.0')
+
+if s:has_lua && !exists('s:lua')
+  lua <<EOF
+  function lsc_last_difference(old, new, offset)
+    local length = math.min(#old, #new)
+    for i = 0, length - 1 do
+      if old[#old - i + offset] ~= new[#new - i + offset] then
+        return -1 * i
+      end
+    end
+    return -1 * length
+  end
+  function lsc_first_difference(old, new, offset)
+    local length = math.min(#old, #new)
+    for i = 1, length do
```

Comment:
Should have considered this zero indexed. Misses changes on the first line.

Reply:
Fixed in https://github.com/natebosch/vim-lsc/commit/974fbe47734f6c6ac8268bc6e2dab8685ffaa5f6